templates/miq-template.yaml

```diff
@@ -88,7 +88,7 @@ objects:
             persistentVolumeClaim:
               claimName: ${NAME}
         containers:
-        - image: fbladilo/miq-app:latest
+        - image: fbladilo/miq-app:${APPLICATION_IMG_TAG}
```

Comment:
@simaishi I tested without it, but OCP does not like it if the 'image' key is missing from container : 
```bash
[root@franco-ocp-eval-master tmp]# oc new-app --template=manageiq 
error: error processing template manageiq: only encoded map or array can be decoded into a struct
```
However , an empty "image" key seems to work fine : 
```bash
[root@franco-ocp-eval-master tmp]# oc export templates manageiq | grep "image:"
          image: ""
          image: fbladilo/miq-memcached:1.4.15
          image: fbladilo/miq-postgresql:9.5
[root@franco-ocp-eval-master tmp]# oc replace -f miq-template.yaml 
template "manageiq" replaced
[root@franco-ocp-eval-master tmp]# oc new-app --template=manageiq 
--> Deploying template manageiq

     manageiq
     ---------
     ManageIQ appliance with persistent storage
```
If it sounds Ok with you I can go ahead and empty all those image keys to avoid confusion. The DC is actually replacing them with the ImageStream tag requested.